draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -894,38 +893,32 @@ Elligator2 can be simplified with projective coordinates.
 
 ((TODO: write this variant))
 
-# Random Oracles
-
-## Interface
+# Random Oracles {#ffstv}
 
-The generic interface for deterministic encoding functions to elliptic curves is as follows:
+Some applications require a Random Oracle (RO) of points, which can be constructed
+from deterministic encoding functions. Farashahi et al. {{FFSTV13}} showed a
+generic mapping construction that is indistinguishable from a random oracle.
+In particular, let `f : {0,1}^* -> E(F)` be a deterministic encoding function,
+and let `H0` and `H1` be two hash functions modeled as random oracles that map
```

Comment:
`H0`, `H1` aren't needed any more, but need to instead define it over `hash2curve` implementations.